dll/win32/shell32/CFolder.cpp

```diff
@@ -1,21 +1,8 @@
 /*
- * Folder implementation
- *
- * Copyright 2015 Mark Jansen
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ * PROJECT:     shell32
+ * LICENSE:     GPL-2.0+ (https://spdx.org/licenses/GPL-2.0+)
```

Comment:
Intentional license change?

Reply:
Yeah, at the time I just slapped a random header on it, but since it incorporates wine code i should maybe stick to gplv2 :(